next/components/forms/segments/AccountSections/MyApplicationsSection/MyApplicationHistory.tsx

```diff
@@ -42,10 +56,21 @@ const MyApplicationHistory = ({ historyData }: MyApplicationHistoryProps) => {
             </span>
           </div>
           <div className="flex flex-col items-center">
-            {historyData?.map((data, i) => (
+            {historyData?.map((data: any, i) => (
               <div className="flex w-full border-t-2 p-4" key={i}>
-                <span className="text-p2 min-w-[240px]">{data?.editDate}</span>
-                <span className="text-p2 w-full">{data?.description}</span>
+                <span className="text-p2 min-w-[240px]">
+                  <FormatDate>
+                    {/* eslint-disable-next-line @typescript-eslint/no-unsafe-member-access */}
+                    {data?.DatumZmeny}
```

Comment:
Check if FormatDate doesn't fail when the date is undefined / null.

Reply:
it doesn't, it'll print "Invalid Date"